team/482451/montyhall.py

```diff
@@ -0,0 +1,30 @@
+from random import randint
+
+c = 0
+list2 = []
```

Comment:
Probably used to store intermediate results? 
No need for list. You can use int to add a win (1) or a loss (0). Then there will be no need for a count function later.